offline/QA/Jet/DijetQA.cc

```diff
@@ -16,6 +16,7 @@
 
 #include <phool/PHCompositeNode.h>
 
+#include <TStyle.h>
```

Comment:
Minor suggestion: a little whitespace to visually offset the includes from the body of the code..
```suggestion
#include <TStyle.h>

```